x-pack/plugins/observability_ai_assistant/server/routes/chat/route.ts

```diff
@@ -40,7 +40,10 @@ const chatRoute = createObservabilityAIAssistantServerRoute({
   handler: async (resources): Promise<Readable> => {
     const { request, params, service } = resources;
 
-    const client = await service.getClient({ request });
+    const [client, cloudStart] = await Promise.all([
+      service.getClient({ request }),
+      resources.plugins.cloud?.start() || Promise.resolve(undefined),
```

Comment:
nit: do you need the `Promise.resolve(undefined)`? 
```suggestion
      resources.plugins.cloud?.start(),
```